src/main.cpp

```diff
@@ -5305,27 +5340,18 @@ std::string GetWarnings(const std::string& strFor)
 //
 
 
-bool static AlreadyHave(const CInv& inv) EXCLUSIVE_LOCKS_REQUIRED(cs_main)
+bool AlreadyHave(const CInv& inv)
```

Comment:
This function is still called with the cs_main lock acquired, consider restoring the attribute

Reply:
Done in main.h